ios/RN/RNCameraManager.m

```diff
@@ -550,6 +550,20 @@ + (NSDictionary *)barcodeDetectorConstants
         }];
 }
 
+RCT_EXPORT_METHOD(hasTorch: (RCTPromiseResolveBlock)resolve
+                  rejecter:(RCTPromiseRejectBlock)reject) {
+    NSArray *devices = [AVCaptureDevice devicesWithMediaType:AVMediaTypeVideo];
```

Comment:
I think `videoCaptureDeviceInput?.device` and `getDevice()` from `RNCamera` should be used to get the device

Reply:
Using this approach we can't call this method when the camera is not mounted, I think it's better to have the information whether the camera is mounted or not.

Reply:
Yes, it would be nice to have a method to detect the feature outside the camera component, but right now it is a method of the camera component. For the iOS implementation the feature detection is done on a specific device and it is best to detect the torch on the device that is currently being used

Reply:
is this method available? I don't find the method in my project.
 version "react-native-camera": "3.43.8"